homeassistant/components/spider/climate.py

```diff
@@ -68,11 +69,14 @@ def device_info(self) -> DeviceInfo:
     @property
     def supported_features(self) -> ClimateEntityFeature:
         """Return the list of supported features."""
+        features = (
+            ClimateEntityFeature.TARGET_TEMPERATURE
+            | ClimateEntityFeature.TURN_OFF
+            | ClimateEntityFeature.TURN_ON
```

Comment:
Don't we need to check if the HAVC modes are actually supported here? It seems like `self.support_hvac` is populated conditionally?